test/controllers/sessions_controller_test.rb

```diff
@@ -13,7 +13,7 @@ class SessionsControllerTest < ActionDispatch::IntegrationTest
   end
 
   test "should get new" do
-    get sign_in_url
+    get sign_in_url, headers: {"Turbo-Frame" => "modal"}
```

Comment:
I think this test is no longer really doing what it says. This redirects to the root_path now which yields a success, but it's not the sessions#new page.

Is there a way to open the sessions#new modal after redirect to the home page?

Reply:
When setting the headers with a turbo frame modal we do render the #new view. Maybe this test should go a bit further and ensure that we render this view. Not just an ok

Reply:
I have updated the test suite to cover both cases